src/Gfx/include/OpenLoco/Gfx/PngImage.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <vector>
+
+#include <filesystem>
```

Comment:
with filesystem being such an expensive header i've usually been changing the interfaces to take a std::string (u8string). Although probably fine here.